lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -215,45 +202,3 @@ function dosomething_northstar_build_ns_user($user, $form_state) {
   // Return fully built northstar user object.
   return $ns_user;
 }
-
-
-class Northstar {
```

Comment:
I'd like to make an actual PHP library that we could share between projects, so let's clear this up for now (since it's currently just squatting on the global `Northstar` namespace).